pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -60,6 +60,10 @@ func validateInstanceSpec(spec *sc.InstanceSpec, fldPath *field.Path, create boo
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("planName"), spec.PlanName, msg))
 	}
 
+	if spec.Parameters != nil {
+		// TODO: validate parameters
```

Comment:
@nilebox are you going to do this in a follow-up? If so, my nit is: can you create an issue for that if you haven't already?